src/MSSQL/CSV/Preprocessor.php

```diff
@@ -36,7 +37,7 @@ public function setDelimiter($delimiter)
      * @return string output filename
      * @throws \Exception
```

Comment:
už to tady zas vidim :) Ale nakonec jsme to nerozhodli. https://keboola.slack.com/archives/C9FEAHW7N/p1526986163000078